README.md

```diff
@@ -161,3 +161,52 @@ There are three ways of creating a template.
     ![Image of default template](https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/_assets/SavedList.png)
 
 6. Modify report as you wish and click "Advanced Editor" button from the menu. Copy all contents and create a file like "your custom template name.template". Please make sure file name ends with '.workbook'.
+
+## How to associate any existing template to new category in Workbooks
```

Comment:
should this be "Additional categories" ?  this allows you to put workbooks in a whole bunch of places, right, not just 2?